project/Dependencies.scala

```diff
@@ -67,23 +67,24 @@ object WellcomeDependencies {
 
 object ExternalDependencies {
   lazy val versions = new {
-    val akka = "2.6.9"
+    val akka = "2.6.10"
     val akkaHttp = "10.1.11"
     val akkaHttpCirce = "1.32.0"
     val akkaStreamAlpakka = "1.1.2"
     val apacheCommons = "3.7"
     val apacheLogging = "2.8.2"
     val aws = "1.11.504"
     val circe = "0.13.0"
-    val elastic4s = "7.9.0"
+    val elastic4s = "7.9.2"
     val fastparse = "2.3.0"
     val swagger = "2.0.10"
     val mockito = "1.9.5"
-    val scalatestplusMockito = "3.1.0.0"
+    val scalatest = "3.2.3"
+    val scalatestPlus = "3.1.2.0"
```

Comment:
Could we keep this as `scalatestplus`? `scalatestPlusMockitoArtifactId` made me think it was `scalatestAndMockitoArtifactId`.

Reply:
Done.